src/utils/errorHandler.js

```diff
@@ -0,0 +1,13 @@
+import { t } from 'i18next';
+
+export const errorApiHandler = response => {
+  let error = t('Errors:apiError');
+
+  const { data } = response;
```

Comment:
you dont have `data` if the network fails (`NETWORK_ERROR`) by default or there is a `CLIENT_ERROR`